gitlint/__init__.py

```diff
@@ -163,6 +163,22 @@ def get_vcs_root():
     return (None, None)
 
 
+def processFile(filename, modified_files, commit, force, gitlint_config):
+    vcs, _ = get_vcs_root()
```

Comment:
This should be called only once, as it spawns two processes.

Reply:
Fixed.